assets/translations/es-CL/postSurveyMessages.private.json

```diff
@@ -1,4 +1,4 @@
 {
   "triggerMessage": "Antes de terminar, ¿Estarías dispuesto a responder algunas preguntas sobre el servicio que recibiste hoy? Por favor, responda Sí o No.",
-  "postSurvetCompleteMessage": "Gracias por contactarnos. Por favor, contáctanos nuevamente si necesita más ayuda."
+  "postSurveyCompleteMessage": "Gracias por contactarnos. Por favor, contáctanos nuevamente si necesitaa más ayuda."
```

Comment:
Just double check the word `necesitaa`, there's a double `a`.

Reply:
@murilovmachado  - Done.